package.json

```diff
@@ -8,7 +8,8 @@
     "node_modules"
   ],
   "devDependencies": {
-    "generator-jhipster": "5.3.4"
+    "generator-jhipster": "6.5.1",
+    @openapitools/openapi-generator-cli": "0.0.14-4.0.2"
```

Comment:
please check if Json is correct here (`@openapitools` does not start from the quote)

Reply:
fixed